Makefile.nxdk

```diff
@@ -0,0 +1,16 @@
+XBE_TITLE = ZombieBreakoutX
```

Comment:
nit: This wouldn't look nice in the games list; I'd use "Zombie-Breakout"; the X suffix/prefix is a stupid meme anyway.